luigi/execution_summary.py

```diff
@@ -416,10 +486,12 @@ def _summary_wrap(str_output):
     """).format(str_output=str_output)
 
 
-def summary(worker):
+def summary(worker, summary_dict=None):
```

Comment:
Let's not change this public function , Just use the helper methods in the init function.

Reply:
Done! 👍